tournaments/index.js

```diff
@@ -960,6 +960,7 @@ class Tournament {
 					Economy.logTransaction(Chat.escapeHTML(winner) + ' has won ' + firstMoney + ' ' + (firstMoney === 1 ? global.currencyName : global.currencyPlural) + ' from a tournament.');
 				});
 			});
+			SG.addExp(wid, this.room, Math.round(tourSize));
```

Comment:
wid works i tested that before commiting this. rid however does not because the entire runnerup area fails so its never even used